### PR TITLE
Patch for #55 (missing dependency hr-timesheet)

### PR DESCRIPTION
**Pull request by katyukha**
Patch for #55 

Added dependency to _.travis.yml_
`IOError: File not found: account_analytic_analysis_recurring/` bug seems to be fixed

### CODE REVIEW

File: project_sla/m2m.py [hunk 1]
```diff
@@ -72,4 +72,4 @@ def replace(ids):
     assert unlink(99) == [(3, 99)]
     assert clear() == [(5, 0)]
     assert replace([97, 98, 99]) == [(6, 0, [97, 98, 99])]
-    print("Done!")
+    print("Done!")  # pylint: disable=print-statement,superfluous-parens
```
Comment (pedrobaeza) [inline project_sla/m2m.py]: You can remove entirely this line.

Comment (katyukha) [inline project_sla/m2m.py]: removed!

**Comment by pedrobaeza** [comment]
Well, there are still some issues to greenify the branch, but the issue commented is solved, so :+1:

**Comment by tafaRU** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:, but this branch build log is awfull :ambulance:

**Pull request merged**